client/my-sites/checkout/composite-checkout/components/checkout-help-link.tsx

```diff
@@ -173,11 +173,7 @@ export default function CheckoutHelpLink() {
 		};
 	} );
 
-	const userAllowedToHelpCenter = !! (
-		userId &&
-		config.isEnabled( 'calypso/help-center' ) &&
-		shouldShowHelpCenterToUser( userId )
-	);
+	const userAllowedToHelpCenter = !! ( userId && config.isEnabled( 'calypso/help-center' ) );
```

Comment:
I think `userId &&` was only added as a type guard for `shouldShowHelpCenterToUser`

Reply:
Thanks for pointing that out. I noticed that too and forgot to leave a note. My thinking was "what if a non-null `userId` means this was acting as an implicit is-the-user-logged-in check?". But in the context of checkout we should have the user always logged-in. Plus, even if not Help Center should work better for them anyway (including the fact that Zendesk Messaging supports logged-out users).